examples/hardware_test.rs

```diff
@@ -1,21 +1,126 @@
 #![feature(alloc)]
 #![no_std]
-
+/// Hardware regression tests.
+/// Need P0.03 and P0.04 to be connected (on a nrf52-dk).
+#[macro_use]
 extern crate alloc;
 
 use alloc::string::String;
 use tock::console::Console;
+use tock::gpio::{GpioPinUnitialized, InputMode};
+use tock::syscalls;
 use tock::timer;
 use tock::timer::Duration;
 
+static mut STATIC: usize = 0;
```

Comment:
I would prefer I you could replace this `static mut usize` with [AtomicUsize](https://doc.rust-lang.org/std/sync/atomic/struct.AtomicUsize.html) to get rid of the `unsafe`

Reply:
It was exactly my point to show that static muts work.